include/deal.II/lac/parallel_vector.h

```diff
@@ -1257,7 +1257,7 @@ namespace parallel
     Vector<Number> &
     Vector<Number>::operator = (const Vector<Number> &c)
     {
-      return this->template operator=<Number>(c);
+      return this->operator=<Number>(c);
```

Comment:
The `template` needs to be there in C++98. It may be true that it can be omitted in C++11. I'd vote for a `#ifdef _MSC_VER` in this case if necessary.